app/components/submission/ManuscriptMetadata/SubjectAreaDropdown.js

```diff
@@ -0,0 +1,165 @@
+    label: 'Developmental Biology and Stem Cells',
+  },
+  { value: 'ecology', label: 'Ecology' },
+  {
+    value: 'epidemiology-global-health',
+    label: 'Epidemiology and Global Health',
+  },
+  { value: 'evolutionary-biology', label: 'Evolutionary Biology' },
+  { value: 'genetics-genomics', label: 'Genetics and Genomics' },
+  {
+    value: 'human-biology-medicine',
+    label: 'Human Biology and Medicine',
+  },
+  {
+    value: 'immunology-inflammation',
+    label: 'Immunology and Inflammation',
+  },
+  {
+    value: 'microbiology-infectious-disease',
+    label: 'Microbiology and Infectious Disease',
+  },
+  { value: 'neuroscience', label: 'Neuroscience' },
+  { value: 'physics-living-systems', label: 'Physics of Living Systems' },
+  { value: 'plant-biology', label: 'Plant Biology' },
+  {
+    value: 'structural-biology-molecular-biophysics',
+    label: 'Structural Biology and Molecular Biophysics',
+  },
+]
+
+const findOptionByValue = (value, options) =>
+  options.find(option => option.value === value)
+
+export class ThemelessSubjectAreaDropdown extends React.Component {
+  constructor(props) {
+    super(props)
+
+    const savedOptions = this.props.savedValues.map(value =>
+      findOptionByValue(value, eLifeOptions),
+    )
+
+    this.state = {
+      selectedOptions: savedOptions,
+      hasReachedMultiselectLimit: false,
+    }
+
+    this.handleChange = this.handleChange.bind(this)
+    this.handleBlur = this.handleBlur.bind(this)
+
+    this.customReactSelectStyles = {
+      multiValue: (base, state) => ({
+        ...base,
+        backgroundColor: this.props.theme.colorPrimary,
+      }),
+    }
+  }
+
+  // https://github.com/JedWatson/react-select/blob/v2/src/Select.js
+  // react-select exposes onChange: (ValueType, ActionMeta) => void
+  handleChange = selectedOptions => {
+    if (selectedOptions.length <= 2) {
+      this.setState({ selectedOptions, hasReachedMultiselectLimit: false })
+      if (this.props.onChange) this.props.onChange(selectedOptions)
+    }
+    if (selectedOptions.length >= 2) {
+      this.setState({ hasReachedMultiselectLimit: true })
+    }
+  }
+
+  handleBlur = e => {
+    if (this.props.onBlur) this.props.onBlur(e)
+  }
+
+  render() {
+    const { selectedOptions, hasReachedMultiselectLimit } = this.state
+    const { label, name } = this.props
+
+    const selectChildProps = {
+      components: { ClearIndicator: null },
+      inputId: 'subject-area-select',
+      isMulti: true,
+      name,
+      onBlur: this.handleBlur,
+      onChange: this.handleChange,
+      options: eLifeOptions,
+      styles: this.customReactSelectStyles,
+      value: selectedOptions,
+      filterOption: (option, rawInput) => {
+        const input = rawInput.replace(/^\s+|\s+$/g, '').toLowerCase()
+        const candidate = `${option.label} ${option.value}`
+          .replace(/^\s+|\s+$/g, '')
+          .toLowerCase()
+
+        return candidate.substr(0, input.length) === input
```

Comment:
https://www.w3schools.com/jsref/jsref_startswith.asp

Reply:
Seems like Microsoft Edge doesn't support this: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/startsWith